core/src/lib.rs

```diff
@@ -844,6 +858,42 @@ pub fn get_usage(config: &Config) -> Result<Vec<FileUsage>, Error<GetUsageError>
         })
 }
 
+pub const BYTE: u64 = 1;
+pub const KILOBYTE: u64 = BYTE * 1000;
+pub const MEGABYTE: u64 = KILOBYTE * 1000;
+pub const GIGABYTE: u64 = MEGABYTE * 1000;
+pub const TERABYTE: u64 = GIGABYTE * 1000;
+
+pub const KILOBYTE_PLUS_ONE: u64 = KILOBYTE + 1;
+pub const MEGABYTE_PLUS_ONE: u64 = MEGABYTE + 1;
+pub const GIGABYTE_PLUS_ONE: u64 = GIGABYTE + 1;
+pub const TERABYTE_PLUS_ONE: u64 = TERABYTE + 1;
+
+pub fn calculate_usage(config: &Config, exact: bool) -> Result<String, Error<GetUsageError>> {
```

Comment:
You should move this out of this file, this file has a very narrow purpose. In fact the other call probably shouldn't be here either. I would create a `usage_service` and move the bodies of this function and the other usage function to those places.